Components/Linux/wordpress-recipes-linux/install_wordpress_with_db_pma.yml

```diff
@@ -0,0 +1,124 @@
+name: InstallWordPressWithDbPma
+description: this document installs Wordpress with Db on Amazon Linux
```

Comment:
`This document installs WordPress with a database and phpMyAdmin on Amazon Linux 2`